Content.Shared/Item/ItemComponent.cs

```diff
@@ -54,6 +54,12 @@ public sealed partial class ItemComponent : Component
     /// </summary>
     [DataField, AutoNetworkedField]
     public float StoredRotation = 0;
+
+    /// <summary>
+    /// An additional offset, in pixels, applied to the visual depiction of the item when displayed in the storage UI.
+    /// </summary>
+    [DataField, AutoNetworkedField]
+    public Vector2i StoredOffset;
 }
```

Comment:
Why wouldn't the sprite be updated instead?

Reply:
Because the sprite is also used in the hand, where you might want it towards the bottom of the square amount of space you have while in the inventory it might need to fit any shape.

I just assumed that the plate sprites and their in-hand positioning were intentional.

Reply:
I think having an offset is fine since ideally we limit the extra sprite states (due to maintenance reasons) and just use rotation and offsets whenever possible.